adding new transformation tools blog post

bfitzpat opened the pull request.

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+= So You Want to Transform Data? Fuse Transformation tooling, part 1
+
```
remove this line as otherwise the :attributes: are not applied.

maxandersen commented on _data/team/bfitzpat.adoc:
```diff
@@ -0,0 +1,5 @@
+= Brian "Fitz" Fitzpatrick
+:page-photo_64px: https://static.jboss.org/developer/people/bfitzpat/avatar/64.png
+:page-photo_32px: https://static.jboss.org/developer/people/bfitzpat/avatar/32.png
```
nice mountain :)

how about a smiley face ? :)

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+
+The last transformation tooling we had in JBoss Tools leveraged Smooks
+to convert one format to another. This time we're using the Dozer
+framework, which offers a great base for handling Java to Java
```
this intro is covering a lot of techs - maybe add links to some o fthem ? like Dozer and Fuse ?

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+
+The new tooling appears in the JBoss Tools Integration Stack "Early
+Access" site, so be sure to keep up with that effort. (You can see
+Paul's latest post here: "JBoss Tools Integration Stack - Luna
```
If you write it as http://tools.jboss.org/blog/2015-04-16-JBTIS-421-Final.html["JBoss Tools Integration Stack - Luna integration tooling for BRMS/BPMS, Data Virt, SOA 5.x + Early Access Fuse and SwitchYard."] it shows up a bit clearer as a direct link (and yeah, need to tell Paul to hold back on the long acronym titles ;)

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+
+link:http://tools.jboss.org/blog/2015-04-16-JBTIS-421-Final.html[]
+
+Keep up to date with JBTIS to ensure you have the latest version of the
```
JBTIS is just the jira key - lets call it JBoss Tools Integration Stack or Integration Stack for short.

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+Transformation->Data Transformation item from the palette onto the
+canvas (see below)
+
+image:images/transformation_tools/launching-the-wizard.png[image]
```
not sure if we have a bug but for now use the absolute relative path to get the image to show up  right.

In this case:
image:/blog/images/transformation_tools/launching-the-wizard.png[]

And leave [] empty if you can't come up with an alternate descriptive text.

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+adding this transformation to. In the case below, we have selected the
+Spring Camel configuration in the starter project.
+
+image:images/transformation_tools/new-fuse-transformation-wizard-start-page.png[image]
```
same issue wtih image link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+Today we're focused on converting from XML to JSON, so we'll select
+those two types and click Next.
+
+image:images/transformation_tools/new-fuse-transformation-wizard-selecting-types-to-transform.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+continue. If you have a particularly complex schema, you may need to
+select from the root elements you wish to key on for the transformation.
+
+image:images/transformation_tools/new-fuse-transformation-wizard-source-xml-details.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+"..." button to browse for our file, select xyz-order-schema.json, and
+click Finish to continue.
+
+image:images/transformation_tools/new-fuse-transformation-wizard-target-json-details.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+For now, we'll close our new Transformation and go back to the Camel
+route to finish hooking up the new endpoint.
+
+image:images/transformation_tools/finishing-up.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+the Delete key on your keyboard or right-click on the arrow and select
+"Remove." When it asks if you want to remove the element, click Yes.
+
+image:images/transformation_tools/going-back-to-the-camel-editor.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+. Next, save the Camel editor and it will refresh the view to clean up
+your route a little.
+
+image:images/transformation_tools/wiring-up-our-new-endpoint.png[image]
```
bad link

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+Next time we'll start doing the actual editing of the Transformation
+itself!
+
+image:images/transformation_tools/end-for-today.png[image]
```
bad link

maxandersen commented:
just a quick read through before I crash ;)

Looks great. only nit is the links not rendering when site generated but you can fix those by using link format I commented on.

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,215 @@
+
+image:images/transformation_tools/end-for-today.png[image]
+
+Thanks for reading!
```
we tend to signoff blogs with our name(s) - in case reader forgot who was listed at top ;)

bfitzpat commented:
Ok Max, I've updated the PR - can you take another look to see if I've corrected all the nits? With the updated image links, the images don't show up in the asciidoc.js previewer on Chrome, so I hope those work when it gets uploaded to the site itself.

maxandersen commented:
This looks good - if any issues publishing them i'll figure it out ;) I'll push this live on monday so it gets most exposure.

bfitzpat commented:
Thanks Max!!

maxandersen commented on blog/2015-05-27-so-you-want-to-transform-part-1.adoc:
```diff
@@ -0,0 +1,211 @@
+:page-layout: blog
```
you should just delete the empty space, not the title ;) I've added it in before I pushed.

maxandersen commented:
fixed date and title and then pushed live this morning
